src/components/preview_component/preview.sh

```diff
@@ -9,8 +9,14 @@
 #   run_preview
 run_preview() {
     if ! "$isAWS"; then
-	# Load environment with modules for running GEOS-Chem Classic
-        source ${GEOSChemEnv}
+	if [ ! -f "${InversionPath}/${GEOSChemEnv}" ]; then
```

Comment:
I wonder if this check should be in run_imi.sh since we would want this behavior not just for the preview, but anywhere.

Reply:
These lines have been moved to run_imi.sh in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/9629c549521385b0709e1aa33c1c158082f6ddc7.